core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -513,7 +513,7 @@ class EMConfig {
         }
     }
 
-    fun shouldGenerateSqlData() = isMIO() && (generateSqlDataWithDSE || generateSqlDataWithSearch)
+    fun shouldGenerateSqlData() = generateSqlDataWithDSE || generateSqlDataWithSearch
```

Comment:
why removing the check `isMIO`?

Reply:
thanks. this is a mistake due to the merge